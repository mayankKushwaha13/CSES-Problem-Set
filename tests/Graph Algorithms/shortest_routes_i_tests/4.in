10 20
6 3 5
7 5 8
5 1 8
8 9 5
5 7 10
7 8 9
4 5 8
2 1 10
6 7 4
5 6 3
9 10 9
10 6 4
10 9 9
2 3 4
10 5 9
1 2 5
9 8 3
3 4 9
4 9 9
3 6 1
