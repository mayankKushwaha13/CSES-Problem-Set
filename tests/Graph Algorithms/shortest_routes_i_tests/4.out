0 5 9 18 22 10 14 23 27 36 
